src/parser/jobs/drk/modules/MultiHitSkills.ts

```diff
@@ -1,23 +1,27 @@
-import ACTIONS from 'data/ACTIONS'
 import {AoEUsages} from 'parser/core/modules/AoEUsages'
 
 export class MultiHitSkills extends AoEUsages {
-	suggestionIcon = ACTIONS.QUIETUS.icon
+	suggestionIcon = this.data.actions.QUIETUS.icon
 
 	trackedActions = [
 		{
-			aoeAction: ACTIONS.FLOOD_OF_SHADOW,
-			stActions: [ACTIONS.EDGE_OF_SHADOW],
-			minTargets: 2,
+			aoeAction: this.data.actions.FLOOD_OF_SHADOW,
+			stActions: [this.data.actions.EDGE_OF_SHADOW],
+			minTargets: 3,
 		},
 		{
-			aoeAction: ACTIONS.QUIETUS,
-			stActions: [ACTIONS.BLOODSPILLER],
+			aoeAction: this.data.actions.QUIETUS,
+			stActions: [this.data.actions.BLOODSPILLER],
 			minTargets: 3,
 		},
 		{
-			aoeAction: ACTIONS.UNLEASH,
-			stActions: [ACTIONS.HARD_SLASH],
+			aoeAction: this.data.actions.ABYSSAL_DRAIN,
+			stActions: [this.data.actions.CARVE_AND_SPIT],
+			minTargets: 4,
+		},
+		{
+			aoeAction: this.data.actions.UNLEASH,
+			stActions: [this.data.actions.HARD_SLASH],
```

Comment:
I believe this is 3 now as well. 1-2-3 combo is 770 potency, AoE combo is 290, so you'd need 3 targets to overtake it.

Reply:
1-2-3 combo is 770 potency (plus 600MP and 20 blood) over 3 hits, while AoE is 290 per target (plus 600MP and 20 blood) over 2 hits. 

So over 6 hits, you've made 1540 potency + 1200 MP + 40 blood on the single target vs 870 potency per target + 1800 MP + 60 blood on the AoE, making 2 the appropriate break point.